barretenberg/cpp/pil/vm2/constants_gen.pil

```diff
@@ -91,6 +97,9 @@ namespace constants;
     pol AVM_NUM_PUBLIC_INPUT_COLUMNS = 4;
     pol AVM_PUBLIC_INPUTS_COLUMNS_COMBINED_LENGTH = 2724;
     pol AVM_PC_SIZE_IN_BITS = 32;
+    pol AVM_WRITTEN_PUBLIC_DATA_SLOTS_TREE_HEIGHT = 6;
+    pol AVM_WRITTEN_PUBLIC_DATA_SLOTS_TREE_INITIAL_ROOT = 18291678969210913367302010540259942201271604198321103848479209155223586227821;
```

Comment:
Where do we plan to inject this value into the circuit? Would it be in `precomputed` in the first row?